src/gui/transferlistmodel.cpp

```diff
@@ -699,62 +697,47 @@ QIcon getIconByState(const BitTorrent::TorrentState state)
 
 QColor getDefaultColorByState(const BitTorrent::TorrentState state)
 {
-    // Color names taken from http://cloford.com/resources/colours/500col.htm
-    bool dark = isDarkTheme();
+    // Colors were selected by using https://doc.qt.io/qt-6/qcolorconstants.html#svg-colors
 
     switch (state)
     {
     case BitTorrent::TorrentState::Downloading:
     case BitTorrent::TorrentState::ForcedDownloading:
     case BitTorrent::TorrentState::DownloadingMetadata:
     case BitTorrent::TorrentState::ForcedDownloadingMetadata:
-        if (!dark)
-            return {34, 139, 34}; // Forest Green
-        else
-            return {50, 205, 50}; // Lime Green
+        return {UIThemeManager::instance()->getColor(QLatin1String(), QColorConstants::Svg::green)};
```

Comment:
The code looks suspicious... try this:
```suggestion
        return QColorConstants::Svg::green;
```
The same to all other similar instances below.